plugins/with-mysql/src/index.ts

```diff
@@ -3,7 +3,7 @@
  */
 import { config } from 'dotenv';
 import Queries from './queries';
-import { ServerConfig } from '@build-tracker/server/src/server';
```

Comment:
There is no declaration file for this, so I was getting a type error. I was able to fix it by shortening the import ID.